migaloo/assetlist.json

```diff
@@ -647,6 +647,54 @@
       "logo_URIs": {
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/wind.png"
       }
+    },
+    {
+      "denom_units": [
+        {
+          "denom": "ibc/0E1B883A15D5FCA533332CBAB1A672934C3936920399F2A3EB4F438E3EBAD0E9",
+          "exponent": 0,
+          "aliases": [
+            "factory/osmo1q77cw0mmlluxu0wr29fcdd0tdnh78gzhkvhe4n6ulal9qvrtu43qtd0nh8/shitmos"
+          ]
+        },
+        {
+          "denom": "shitmos",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/0E1B883A15D5FCA533332CBAB1A672934C3936920399F2A3EB4F438E3EBAD0E9",
+      "name": "Shitmos",
+      "display": "shitmos",
+      "symbol": "SHITMOS",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "osmosis",
+            "base_denom": "factory/osmo1q77cw0mmlluxu0wr29fcdd0tdnh78gzhkvhe4n6ulal9qvrtu43qtd0nh8/shitmos",
+            "channel_id": "channel-642"
+          },
+          "chain": {
+            "channel_id": "channel-5",
+            "path": "transfer/channel-5/factory/osmo1q77cw0mmlluxu0wr29fcdd0tdnh78gzhkvhe4n6ulal9qvrtu43qtd0nh8/shitmos"
+          }
+        }
+      ],
+      "images": [
+        {
+          "image_sync": {
+            "chain_name": "osmosis",
+            "base_denom": "factory/osmo1q77cw0mmlluxu0wr29fcdd0tdnh78gzhkvhe4n6ulal9qvrtu43qtd0nh8/shitmos"
+          },
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/migaloo/images/shitmos.png",
```

Comment:
The logo images for assets should be kept in the /images/ directory corresponding to their original chain (in this case, in osmosis' /images directory). There is already a square image in osmosis' /images directory, https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/SHITMOS.png, so we don't need really these. But if you'd like to add a round version (because the original is square), you can do so by adding another object into the `images` array for the original osmosis asset.

Reply:
I moved the images to the osmosis subdirectory, and changed the path to the images to properly refer to those images in the osmosis subdirectory.